slices/bind9-dnsutils.yaml

```diff
@@ -0,0 +1,24 @@
+package: bind9-dnsutils
+
+essential:
+  - bind9-dnsutils_copyright
+
+slices:
+  bins:
+    essential:
+      - bind9-host_bins
+      - bind9-libs_libs
+      - libc6_libs
+      - libedit2_libs
+      - libidn2-0_libs
+      - libkrb5-3_libs
+    contents:
+      /usr/bin/delv:
+      /usr/bin/dig:
+      /usr/bin/mdig:
+      /usr/bin/nslookup:
+      /usr/bin/nsupdate:
+
+  copyright:
+    contents: 
```

Comment:
There appears to be a trailing space character at the end of this `contents:` line, can you remedy?